src/main/scala/com/malliina/push/apns/confs.scala

```diff
@@ -20,7 +22,7 @@ case class KeyId(id: String) extends AnyVal {
 /**
   * @param privateKey downloadable from Apple's developer website
   */
-case class APNSTokenConf(privateKey: Path, keyId: KeyId, teamId: TeamId)
+case class APNSTokenConf(privateKey: BufferedSource, keyId: KeyId, teamId: TeamId)
```

Comment:
It's unfortunate that `BufferedSource` is a closeable resource, but it's not clear who closes it or when it's closed. In practice, `APNSTokenClient` consumes and closes it in its constructor, but to improve clarity I suggest the following changes:

- Change the type of `privateKey` here to PKCS8EncodedKeySpec.
- Move the code that instantiates `PKCS8EncodedKeySpec` (from a resource or a file) to the companion object of `APNSTokenConf` below, such that the stream is immediately closed after consumption: this is done in `APNSTokenClient` at https://github.com/malliina/mobile-push/blob/3b227ec054274dcc686b5683cf6731d079fe0385/src/main/scala/com/malliina/push/apns/APNSTokenClient.scala#L41 but that part can be moved here instead.

This change would avoid `APNSTokenConf` having any open resources in its parameter list.